content/blog/2023/when-to-choose-a-native-mobile-app.md

```diff
@@ -122,42 +139,44 @@ Native mobile apps require a unique codebase for each platform. For example, if
 
 ### Lengthy Development
 *Don't choose a native mobile app when it's a priority to make your app
 available to users quickly.*
 
 Because native mobile apps require a separate codebase for each platform you
 want to support, development generally takes longer than it does to develop a
-single codebase for an RWA. Additionally, each app store has its own set of
-requirements in order to publish an app, and these can be complex. Wait times
-for app store approval to publish or update the app can be lengthy as well.
+single codebase for a responsive web app. Additionally, each app store has its
+own set of requirements in order to publish an app, and these can be complex.
+Wait times for app store approval to publish or update the app can be lengthy as
+well.
 
 ### Not Discoverable Via Search Engines
 *Don't choose a native mobile app when the people you want to reach will look
 for your content with a search engine.*
 
-Unlike for RWAs, search engines don't index native mobile apps. If your research
-indicates that people frequently use search engines to search for the type of
-content you offer, an RWA may be a better choice. Alternatively, you could build
-a marketing website for search engines to find, which simply introduces your app
-and links people to the appropriate app store for download. (A separate
-marketing website will further increase the costs and development costs.)
+Unlike for responsive web apps, search engines don't index native mobile apps.
+If your research indicates that people frequently use search engines to search
+for the type of content you offer, an RWA may be a better choice. Alternatively,
+you could build a marketing website for search engines to find, which simply
+introduces your app and links people to the appropriate app store for download.
+(A separate marketing website will further increase the design and development
+costs.)
 
 ### Limited Reach
 *Don't choose a native mobile app when you want to reach a broad audience across
 devices.*
 
 Because it takes longer and costs more to reach the same range of people with a
-native mobile app than with an RWA, and because native mobile apps offer less
-backward compatibility, the number of people who can access a native mobile app
-will likely always be more limited than the number of people who can access an
-RWA. Additionally, a native mobile app is limited to mobile devices. To provide
-access to your content on a laptop or desktop, you would need to develop
-separate native desktop software.
+native mobile app than with a responsive web app, and because native mobile apps
+offer less backward compatibility, the number of people who can access a native
+mobile app will likely always be more limited than the number of people who can
+access a responsive web app. Additionally, a native mobile app is limited to
+mobile devices. To provide access to your content on a laptop or desktop, you
+would need to develop separate native desktop software.
```

Comment:
A native iOS app _can_ run on newer Macs- https://support.apple.com/guide/app-store/iphone-ipad-apps-mac-apple-silicon-fird2c7092da/mac

An alternative to native desktop software would be an RWA.